src/GeneralStateTestsFiller/stCreateTest/CREATE_HighNonceDelegatecallFiller.yml

```diff
@@ -0,0 +1,187 @@
+    currentGasLimit: '89128960'
+    currentBaseFee: '10'
+    currentNumber: '1'
+    currentTimestamp: '1000'
+    previousHash: 5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6
+
+  pre:
+    a94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+      balance: 1000000000
+      code: ''
+      nonce: 0
+      storage: {}
+
+    000000000000000000000000fffffffffffffffe:
+      balance: 0
+      code: |
+        :yul
+        {
+          // initcode: { return(0, 1) }
+          mstore(0, 0x60016000f3000000000000000000000000000000000000000000000000000000)
+          mstore(0, create(0, 0, 5))
+          return(0, 32)
+        }
+      nonce: '0xfffffffffffffffe'
+      storage: {}
+
+    000000000000000000000000ffffffffffffffff:
+      balance: 0
+      code: |
+        :yul
+        {
+          // initcode: { return(0, 1) }
+          mstore(0, 0x60016000f3000000000000000000000000000000000000000000000000000000)
+          mstore(0, create(0, 0, 5))
+          return(0, 32)
+        }
+      nonce: '0xffffffffffffffff'
+      storage: {}
+
+    b94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+      balance: 0
+      code: |
+        :yul
+        {
+          let calltype      := calldataload(4)
+          let callcontract  := calldataload(36)
+          let callcount     := calldataload(68)
+
+          if eq(calltype, 0) {
+            for { let i := 0 } lt(i, callcount) { i := add(i, 1) } {
+              pop(delegatecall(sub(gas(), 1000), callcontract, 0, 0, 0, 32))
+              let result := mload(0)
+              sstore(i, result)
+            }
+          }
+          if eq(calltype, 1) {
+            for { let i := 0 } lt(i, callcount) { i := add(i, 1) } {
+              pop(callcode(sub(gas(), 1000), callcontract, 0, 0, 0, 0, 32))
+              let result := mload(0)
+              sstore(i, result)
+            }
+          }
+          if eq(calltype, 2) {
+            for { let i := 0 } lt(i, callcount) { i := add(i, 1) } {
+              pop(call(sub(gas(), 1000), callcontract, 0, 0, 0, 0, 32))
+              let result := mload(0)
+              sstore(i, result)
+              pop(delegatecall(sub(gas(), 1000), callcontract, 0, 0, 0, 32))
+            }
+          }
+          if eq(calltype, 3) {
+            for { let i := 0 } lt(i, callcount) { i := add(i, 1) } {
+              pop(delegatecall(sub(gas(), 1000), callcontract, 0, 0, 0, 32))
+              pop(call(sub(gas(), 1000), callcontract, 0, 0, 0, 0, 32))
+              let result := mload(0)
+              sstore(i, result)
+            }
+          }
+        }
+      nonce: '0xfffffffffffffffe'
+      storage: {}
+
+  transaction:
+    data:
+    # Parameters:
+    #  1) Call type: 0 - delegateCall, 1 - callCode, 2 - call + delegate call, 3 - delegate call + call
+    #  2) Called contract nonce
+    #  3) Contracts to create
+    # - Contract A (Nonce=0xfffffffffffffffe) delegate calls Contract B (Nonce=0xfffffffffffffffe), contract created first call, then fails
```

Comment:
then why it is `a_max_nonce_delegate_b_max_nonce_minus_1`
if nonce is the same

Reply:
Changed labels in latest push.